src/controllers/GraphHelpers.tsx

```diff
@@ -137,13 +138,14 @@ export const transformNodesToGQL = (nodes: any) => {
         gqlNode           = { ...node.data };
         gqlNode.reactNode = node;
         gqlNode.serviceId = node.data.serviceId  // random value for now 
-        // remove allowedConnections
+        // remove unnecessary fields
```

Comment:
I agree...! The old comment was redundant (it's pretty obvious what the next line does) and inconsistent/weird (why didn't all the other deletes get a comment?). What if we take this further and make the new comment even more helpful: instead of calling the fields "unnecessary" with no further explanation, we could state that these fields are not part of the backend's workflownode schema.